developer_manual/digging_deeper/task_processing.rst

```diff
@@ -242,6 +242,11 @@ The task class objects have the following methods available:
  * ``getWebhookUri()`` This returns the webhook URI that will be notified when the task execution has ended
  * ``getWebhookMethod()`` This returns the HTTP method that will be used for the webhook when the task execution has ended
 
+
+.. versionadded:: 33.0.0
+
+ * ``getUserFacingErrorMessage()`` This returns any error message that is meant to be displayed to the user, even if a task has failed, this is not guaranteed to be set.
```

Comment:
maybe it can be clarified here what is returned when it's not set

Reply:
We also don't do that for the other methods :thinking:

Reply:
it might be better to do for all of them but the function signatures/docs can be seen in the IDE too